src/main/ts/components/EditorPropTypes.ts

```diff
@@ -90,6 +90,7 @@ export const EditorPropTypes: IEditorPropTypes = {
   onEditorChange: PropTypes.func,
   outputFormat: PropTypes.oneOf(['html', 'text']),
   value: PropTypes.string,
+  selection: PropTypes.object,
```

Comment:
It might be worth describing this using `PropTypes.oneOfType([...])` and an array `PropTypes.shape({...})` using the interfaces defined in `TinymceConfig`. Should throw warnings/errors then when there's an undefined type being passed through.